src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1499,6 +1500,10 @@ private function getErrorMessages(Exception $e)
                     'Admin.Orderscustomers.Notification'
                 ),
             ],
+            InvalidModuleException::class => $this->trans(
+                'Payment module name is invalid',
+                'Admin.Catalog.Notification'
+            ),
```

Comment:
Do we need this error notification? It is a dropdown selector, right?

Reply:
It's an error in an alert box if the form is submitted and payment module is not right filled.

Reply:
If we want to stay consistent, we should use _Please fill in all the required fields._ - an error notification that already exists on this page. But it looks like all fields are required on this page and therefore there is no red asterisk to let the user know which fields are required. It might be confusing for the users. So I suggest _You must choose a payment module to create the order._ in _Admin.Orderscustomers.Notification_.